myceli.Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Base build stage
```

Comment:
I didn't see at first pass, but can you document how you are invoking this image in the README?  We could setup a docker-compose network and have everything run on it locally, as well as a similar setup for CI tests.